### PR TITLE
Feature/android systembar color

### PR DESCRIPTION
**Pull request by kaiwidmer**
Ticket: https://app.asana.com/0/1199509904376633/1199924775685097/f

### CODE REVIEW

File: app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt [hunk 1]
```diff
@@ -78,4 +86,34 @@ abstract class BaseFragment<B : ViewBinding, V : BaseView> : Fragment(), BaseVie
     internal abstract fun createBinding(): B
 }
 
-interface KeyboardUsingFragment
\ No newline at end of file
+@Suppress("DEPRECATION")
```
Comment (tschuls) [inline app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt:89]: Why do we use this deprecation?

File: app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt [hunk 2]
```diff
@@ -78,4 +86,34 @@ abstract class BaseFragment<B : ViewBinding, V : BaseView> : Fragment(), BaseVie
...
+            } else {
+                window.insetsController?.setSystemBarsAppearance(0, WindowInsetsController.APPEARANCE_LIGHT_NAVIGATION_BARS)
+            }
+        } else {
+            if (systemBarColor.light) {
+                window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
+            } else {
+                window.decorView.systemUiVisibility = 0
+            }
+        }
```
Comment (tschuls) [inline app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt:109]: Could you reduce the complexity by extracting two methods depending on the SDK version?

File: app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt (hunk 1 above)
Comment (kaiwidmer) [reply app/src/main/java/ch/dreipol/rezhycle/fragments/BaseFragment.kt]: Below api 30 there is only the deprecated way for setting light systemBar, as far as I can see

**Review by tschuls — APPROVED** [review]
👍 thanks

**Pull request merged**